packages/story-editor/src/components/panels/design/animation/karma/animation.karma.js

```diff
@@ -170,4 +181,119 @@ describe('Animation Panel', function () {
 
     expect(animationState).toBe(STORY_ANIMATION_STATE.PLAYING_SELECTED);
   });
+
+  it('should render the animation panel with warning if on the first page.', async function () {
+    // add text to the canvas
+    await fixture.editor.library.textTab.click();
+    await fixture.events.click(fixture.editor.library.text.preset('Paragraph'));
+    await waitFor(() => {
+      if (!fixture.editor.canvas.framesLayer.frames[1].node) {
+        throw new Error('node not ready');
+      }
+    });
+
+    // navigate to the animations panel
+    await fixture.events.click(fixture.editor.sidebar.designTab);
+    await fixture.events.click(
+      fixture.editor.sidebar.designPanel.animationSection
+    );
+
+    // A warning should be displayed per AMP spec - animations are disabled on first page
+    const contrastWarning = await fixture.screen.findByTestId(
+      'warningContainer'
+    );
+
+    await expectAsync(contrastWarning).toHaveNoViolations();
+  });
+
+  it('should render the animation panel with effectChooser input disabled if on the first page.', async function () {
+    // add text to the canvas
+    await fixture.editor.library.textTab.click();
+    await fixture.events.click(fixture.editor.library.text.preset('Paragraph'));
+    await waitFor(() => {
+      if (!fixture.editor.canvas.framesLayer.frames[1].node) {
+        throw new Error('node not ready');
+      }
+    });
+
+    // navigate to animations panel
+    await fixture.events.click(fixture.editor.sidebar.designTab);
+    await fixture.events.click(
+      fixture.editor.sidebar.designPanel.animationSection
+    );
+
+    const panel = fixture.editor.sidebar.designPanel.animation;
+    // the effectChooser should be disabled on the first page
+    expect(panel.effectChooser.disabled).toBeTrue();
+  });
+
+  it('should render the animation panel with inputs disabled when page becomes the first page', async function () {
+    const { id: firstPageId } = await getCurrntPage();
```

Comment:
If you correct the above, you'll need to correct the spelling in some of these files toO

Reply:
Copy and paste kept the issue insync 😆 - good catch

Reply:
Haha no problemo!